features/org.wso2.analytics.is.feature/src/main/capp/dashboard/IsAnalytics-Gadget-AlertsSummery_1.0.0/IsAnalytics_Gadget_AlertsSummery/js/main.js

```diff
@@ -92,33 +94,33 @@ $(document).ready(function () {
         }
     });
 
+    function getPdfTableColumns() {
 
-    function getPdfTableColsAndInfo(){
-        this.getPdfTableColumns= function(){
-            return columns.map(function (column){
-                var newColumn = {};
-                newColumn["title"] = column["title"];
-                newColumn["dataKey"] = column["data"];
-                return newColumn;
-            });
-        }
+        return columns.map(function(column) {
 
-        this.getPdfTableInfo = function(){
-            var pdfInfo = {};
-            pdfInfo["title"] = "SECURITY ALERT SUMMERY";
-            pdfInfo["headerInfo"] = "Starting Date   : " + renderDateTime(parseInt(listnedTimeFromValue)) +
-                 "\n\nEnding Date    : " + renderDateTime(parseInt(listnedTimeToValue));
-            pdfInfo["fileName"]="Security Alert Summery Report";
-            return pdfInfo;
-        }
+            var newColumn = {};
+            newColumn["title"] = column["title"];
+            newColumn["dataKey"] = column["data"];
+            return newColumn;
+        });
+    }
 
-        function renderDateTime(data, type, row) {
-            var date = new Date(data);
-            return date.toLocaleString("en-US");
-        }
+    function getPdfTableInfo() {
+
+        var pdfInfo = {};
+        pdfInfo["title"] = "SECURITY ALERTS SUMMERY";
```

Comment:
Thanks for catching this. Other file name and folder names are also with this typo. I think those do not make a much effect on user.